packages/website/docs/SingleSelect.md

```diff
@@ -1,24 +1,44 @@
-## Guidelines
+# Best Practices
 
-### General
+Use a Single select to have users **filter options from a long list and select only one**. A Single select is especially appropriate when the list of available options is very long or when the space is limited.
 
--   If space allows, and there are no more than 7 options to choose from, consider using [radio buttons](#/components/Radio) instead.
--   Keep option labels as short as possible in order to limit width to a minimum.
+List options in alphanumerical order unless a more suited ordering rationale applies, for example when listing size or security level options.
 
-### Options Sorting
+Always include the ability to filter options when the list contains 20 or more.
 
--   Always put the most common items at the top of the list. If no option is more common than the others, use alphanumeric or chronological order.
+## Labeling
 
-### Label
+Keep titles and labels short, preferably **under three words**.
 
--   In most cases, you should include a label to identify the component.
+### Title
+
+The title should indicate the type of information to provide.
+
+Provide a descriptive title without action verbs. For instance, write "Favorite drink" rather than "Select your favorite drink".
+
+### Labels
+
+Labels identify each option and should be self-explanatory. The width of the input should allow to fully display the name of the selected option.
+
+Use a consistent writing style for all options in the list.
+
+## Help Text and Instructions
+
+The placeholder text should indicate the type of information to select. Use an action verb. For example, write "Select a drink" rather than "Select an option".
+
+## Feedback and Validation
+
+Allow the addition of custom values only when it doesn't increase the risk of failure and when there may be options other than those listed.
+
+Examples:
+-   When users select a country: do not allow custom values.
+-   When users select their favorite color: allow custom values.
```

Comment:
```suggestion
-   When users select their favorite color: allow custom values so that users can enter specific shades such as "teal" and "chartreuse".
```